src/vs/monaco.d.ts

```diff
@@ -1678,6 +1678,12 @@ declare namespace monaco.editor {
 		sectionHeaderText?: string | null;
 	}
 
+	export enum TextDirection {
+		LTR = 0,
+
+		RTL = 1,
+	}
```

Comment:
This enum is the same as `SelectionDirection`, but IMO that doesn’t make sense in this context. I could merge them. This means to change this to `SelectionDirection` for backwards compatibility, or to rename `SelectionDirection` to `TextDirection` for a more meaningful variable name.